docker-compose-production.yml

```
@@ -13,6 +13,8 @@ services:
     environment:
       - HOST_JOBS_DIR=${HOME}/pl_ag_jobs
       - NODE_ENV=production
+    extra_hosts:
+      - "host.docker.internal:172.17.0.1"
```

Can we add a comment to help folks navigate the problem that Seth faced in the linked issue? Ditto for the other file.

```suggestion
      # This must be changed if you've changed Docker's address pools.
      - "host.docker.internal:172.17.0.1"
```

---

Thanks for the review. Updated accordingly.